packages/flutter_tools/lib/src/fuchsia/fuchsia_device.dart

```diff
@@ -480,16 +487,22 @@ class FuchsiaDevice extends Device {
 
   @override
   Future<String> get sdkNameAndVersion async {
+    const String defaultName = 'Fuchsia';
+    if (!globals.fuchsiaArtifacts.validateSshConfig()) {
+      globals.printError('Could not determine Fuchsia sdk name or version '
```

Comment:
These could be `printTrace`. If someone is really trying to run on Fuchsia, they'll get a better error message later.

Reply:
Done and done.

Reply:
Errrr - I changed the wrong one, I'll fix that.